ffc/backends/ufc/dofmap_template.py

```diff
@@ -18,6 +18,11 @@
 {tabulate_dofs}
 }}
 
+void tabulate_dof_permutations_{factory_name}(const int64_t* restrict global_indices, int* restrict perm, int ndofs)
```

Comment:
Should we keep following the convention that the output argument is first?

Reply:
Sure. I've now reordered the args. Thanks.